_layouts/default.html

```diff
@@ -20,6 +20,8 @@
 
     <!-- LLNL Analytics -->
     <script src="{{site.baseurl}}/assets/js/matomo.js"></script>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script type="text/javascript" src=""{{site.baseurl}}/assets/js/googleAnalyticsTag.js"></script>
```

Comment:
@lee1043 there seems to be a double "" in there, should be replaced with a single

Reply:
@durack1 thanks, fixed it.